### PR TITLE
refactor(context) : remove callback OR in core

### PR DESCRIPTION
**Pull request by youf-olivier**

### CODE REVIEW

File: packages/core/src/routes/OidcRoutes.js [hunk 1]
```diff
@@ -47,7 +44,7 @@ const OidcRoutes = ({
 
   switch (path) {
     case callbackPath:
-      return <CallbackComponent callbackComponentOverride={callbackComponentOverride} />;
```
Comment (guillaumechervetaxa) [inline packages/core/src/routes/OidcRoutes.js]: nice update @youf-olivier ,
you have to update redux package too.

Comment (youf-olivier) [reply packages/core/src/routes/OidcRoutes.js]: The problem was in the context package only

Comment (guillaume-chervet) [reply packages/core/src/routes/OidcRoutes.js]: May be but thos remove break the feature on redux pacakge :p

Comment (youf-olivier) [reply packages/core/src/routes/OidcRoutes.js]: I Don't think so, the redux package didn't use the callbackComponentOverride prop. Or I missed something else ? 

From oidc.js in Redux Package : 

return (
    <OidcProvider store={store} userManager={getUserManager()}>
      <OidcRoutes
        configuration={configuration}
        notAuthenticated={notAuthenticated}
        notAuthorized={notAuthorized}
        callbackComponent={withComponentOverrideProps(
          AuthenticationCallback,
          callbackComponentOverride
        )}
      >
        {children}
      </OidcRoutes>
    </OidcProvider>
  );

Comment (guillaumechervetaxa) [reply packages/core/src/routes/OidcRoutes.js]: yep

**Pull request merged**